### PR TITLE
Add: 5 questions 

### PR DESCRIPTION
**Pull request by Gautam-Hegde**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Number of questions added in each domain is as follows:
Agile-quiz: 1
Information-Technology-quiz: 1
html-quiz: 1
security-quiz: 2

### CODE REVIEW

File: src/data/information-technology-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const informationTechnologyQuiz = [
+  {
+    Question: "Which of the follwing validates the integrity of data?",
```
Comment (kalashjain23) [inline src/data/information-technology-quiz.ts:3]:
```suggestion
    Question: "Which of the following validates the integrity of data?",
```

File: src/data/html-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,14 @@
 const htmlQuiz = [
+  {
+    Question: "What is the primary purpose of the <canvas> tag?",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:3]:
```suggestion
    Question: "In HTML, what is the primary purpose of the <canvas> tag?",
```

File: src/data/security-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+  },
+{
+    Question:
+      "Which option is an open-source solution to scanning a network for active hosts and open ports?",
+    Answer: "Nmap",
+    Distractor1: "Snort",
+    Distractor2: "Autopsy",
+    Distractor3: "Wireshark",
+    Explanation:
+      "Nmap is a free and open-source network scanner that can be used to scan a network for active hosts and open ports.It is also used to detect the operating system and services running on a host.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:21]:
```suggestion
      "Nmap is a free and open-source network scanner that can be used to scan a network for active hosts and open ports.  It is also used to detect the operating system and services running on a host.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Thank you for your contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by Gautam-Hegde** [comment]
Thanks!
It's encouraging to see new members joining the open source community!
By the way, does this PR have hacktoberfest acceptance?

**Pull request merged**